openreview/conference/builder.py

```diff
@@ -665,8 +695,9 @@ def set_program_chairs(self, emails = []):
         # if first time, add PC console
         if not pcs.web:
             self.webfield_builder.set_program_chair_page(self, pcs)
-        ## Give program chairs admin permissions
+        ## Give program chairs admin permissions and viceversa
         self.__create_group(self.id, '~Super_User1', [self.get_program_chairs_id()])
+        self.__create_group(pcs.id, '~Super_User1', [self.id])
```

Comment:
What is the motivation behind this?

Reply:
some objects have the PC group permission and when the conference group needs to access doesn't have the permission, I'm adding the bidirectional relation so using PC or conference group should be the same.